tests/search/linguistics_tokens/linguistics_tokens.rb

```diff
@@ -0,0 +1,47 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'indexed_search_test'
+
+class DynTeaserArrayTest < IndexedSearchTest
```

Comment:
Please rename to LinguisticsTokensTest (or similar).

Reply:
Renamed.